.github/workflows/lint.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/setup-go@v5
         if: env.GIT_DIFF
         with:
-          go-version: "1.23"
```

Comment:
let's don't change that. We will update to 1.24 once ubuntu and fedora is released.